src/plugins/files_management/public/plugin.ts

```diff
@@ -22,7 +22,15 @@ export class FilesManagementPlugin
       async mount(params: ManagementAppMountParams) {
         const { mountManagementSection } = await import('./mount_management_section');
         const [coreStart, depsStart] = await core.getStartServices();
-        return mountManagementSection(coreStart, depsStart, params);
+
+        const { docTitle } = coreStart.chrome;
+        docTitle.change(PLUGIN_NAME);
+
+        const unmountAppCallback = mountManagementSection(coreStart, depsStart, params);
+        return () => {
+          docTitle.reset();
+          unmountAppCallback();
+        };
```

Comment:
This looks like a problem of the Files Management app not implementing the page breadcrumbs correctly. Based on your screenshot and the markup inspected there, it looks like the screen reader only live region gets its info from the page breadcrumbs.

I'll look into this for this PR, it could e a very small fix :)